src/style-spec/reference/v8.json

```diff
@@ -1857,10 +1863,14 @@
           "android": "7.4.0",
           "ios": "4.10.0",
           "macos": "0.14.0"
+
         }
       },
       "requires": [
-        "text-field"
+        "text-field",
+        {
+          "!" : "text-offset"
```

Comment:
I think this is wrong

Reply:
@ryanhamley say more? this addition seems like something we'd want to keep, no?